vike/shared/page-configs/Config.ts

```diff
@@ -375,6 +381,10 @@ type ConfigBuiltIn = {
    * https://vike.dev/clientRouting#link-prefetching
    */
   prefetchStaticAssets?: PrefetchStaticAssets | ImportString
+
+  /** Modify Timeouts */
+  timeouts?: ConfigTimeout
 }
 type ConfigMeta = Record<string, ConfigDefinition>
 type ImportString = `import:${string}`
+type ConfigTimeout = Record<string, TimeoutDefinition>
```

Comment:
Hi @brillout , should I change `ConfigTimeouts` to `HookTimeouts` too ? 
I kinda a bit confuse and doubt about naming things 😅

Reply:
Sounds good 👍 

Yea, naming is quite a trial-and-error thing. See the mirad of `minor refactor` commits in Vike's `git log` 😅